scripts/check-pr-guardian-contract.sh

```
@@ -40,9 +45,21 @@ executable="$(
 
+if ! bash -n <<<"${executable}"; then
+  echo 'pr-guardian audit reference contains a Bash example that does not parse' >&2
+  exit 1
+fi
+
 for pattern in \
   'while :; do' \
+  "owner='owner'" \
+  "repo='repo'" \
+  'pr_number=123' \
+  'gh pr view "${pr_number}" --repo "${owner}/${repo}"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)